templates/packer/aws/author-publish-dispatcher.json

```diff
@@ -4,6 +4,7 @@
     "aws_subnet_id": "",
```

Comment:
Please add a note to changelog file as per contribution guideline https://shinesolutions.github.io/aem-opencloud/contribution-guideline .